### ISSUE TITLE
some youku video unable to play

### ISSUE THREAD

**Issue by dasteria** [zhuzhuor/Unblock-Youku#432]
video link:http://v.youku.com/v_show/id_XNTg3NjkzMzIw.html?from=y1.6-96.1.1.b5ccaeea35de11e296ac

need to add a new forward rule to cnforward.action

{+forward-override{forward need.a.reliable.proxy:port}}
play.youku.com/play/get.json\?.*

Can this be done?

**Comment by zhuzhuor** [comment]
It still works for me here. Could you try again?


![screen shot 2015-05-28 at 11 52 59 am](https://cloud.githubusercontent.com/assets/928905/7864451/24bb6cba-0530-11e5-83f3-c9a394358257.png)

**Comment by zhuzhuor** [comment]
The URL [has been added](https://github.com/zhuzhuor/Unblock-Youku/commit/d272ab421585706425397e230d81103a85427d19#diff-41aa484bd1cb407067baa0bc2761f236R4) in order to fix another issue for Tudou. See https://github.com/zhuzhuor/Unblock-Youku/issues/434#issuecomment-106167928

Thank you very much!
